kinto/core/testing.py

```diff
@@ -1,34 +1,29 @@
-from collections import defaultdict
-import mock
 import os
-import threading
-import functools
+from collections import defaultdict
 
 try:
     import unittest2 as unittest
-except ImportError:
+except ImportError:  # pragma: no cover
```

Comment:
I think since we are not supporting python2.6 anymore we can probably just import unittest here.